docs/Toy/examples/ast.toy

```diff
@@ -0,0 +1,76 @@
+# RUN: toyc-ch1 %s -emit=ast 2>&1 | FileCheck %s
```

Comment:
These tests are using `toyc-ch1` and `toyc-ch2`, however no such file exist?

Reply:
They don't really use them, I just copied the files including this comment

Reply:
Can you add a note that they are not used for future reference?